boxoffice/views/admin_order.py

```diff
@@ -123,3 +123,60 @@ def admin_org_order(org, order):
     line_items = LineItem.query.filter(LineItem.order == order,
         LineItem.status.in_([LINE_ITEM_STATUS.CONFIRMED, LINE_ITEM_STATUS.CANCELLED])).all()
     return dict(org=org, order=order, line_items=line_items)
+
+
+@app.route('/api/1/organization/<org_name>/order/<int:receipt_no>', methods=['GET'])
+@load_models(
+    (Organization, {'name': 'org_name'}, 'org'),
+    (Order, {'organization': 'org', 'receipt_no': 'receipt_no'}, 'order')
+)
+def order_api(org, order):
+    check_api_access(org.details.get('access_token'))
+
+    line_items_list = []
+    line_items = LineItem.query.filter(LineItem.order == order, LineItem.status.in_([LINE_ITEM_STATUS.CONFIRMED, LINE_ITEM_STATUS.CANCELLED])).all()
+    for line_item in line_items:
+        line_items_list.append({
+            'title': line_item.item.title,
+            'status': LINE_ITEM_STATUS[line_item.status],
+            'base_amount': line_item.base_amount,
+            'discounted_amount': line_item.discounted_amount,
+            'final_amount': line_item.final_amount,
+            })
```

Comment:
This will be more efficient as a list comprehension:

line_items_list = [{
    'title': li.item.title,
    'status': LINE_ITEM_STATUS[li.status],
    'base_amount': li.base_amount,
    'discounted_amount': li.discounted_amount,
    'final_amount': li.final_amount
    } for li in line_items]